src/Components/RateModal.tsx

```diff
@@ -172,9 +178,12 @@ export class RateModal extends Component<IProps, IState> {
 			Platform.OS === 'ios' ?
 				Linking.openURL(iTunesStoreUrl) :
 				Linking.openURL(playStoreUrl);
-        this.setState({ isModalOpen: false });
-        onSendReview();
+		if (onSendReview) {
```

Comment:
@chill-patel Maybe if you separate it in another function you don't have the codeclimate problem

Reply:
Yeah